aim/storage/rockscontainer.py

```diff
@@ -106,12 +107,18 @@ def finalize(self, *, index: Container):
         self._progress_path.unlink()
         self._progress_path = None
 
+        self._db.flush()
```

Comment:
as with the locks, db is considered as 'in progress' unless flushed

Reply:
yup, legit notices, will change ✅